packages/design-system/src/components/tree/tree-node.tsx

```diff
@@ -149,12 +149,15 @@ export type TreeNodeProps<Data extends { id: string }> = {
   level?: number;
   animate?: boolean;
   forceHoverStateAtItem?: string;
+
+  hideRoot?: boolean;
```

Comment:
just a thought: instead of a flag, it could be based on data, if data is array - there is no root, because it's basically the children

Reply:
May do it this way, but this complicates things

Reply:
Will think more about it when I'll be adding more functionality to the list. Will leave as is in this PR.